Main.xaml

```diff
@@ -89,7 +89,7 @@
               <x:Boolean x:Key="IsPinned">False</x:Boolean>
             </scg:Dictionary>
           </sap:WorkflowViewStateService.ViewState>
-          <ui:LogMessage DisplayName="Log Message - Main" sap:VirtualizedContainerService.HintSize="334,91" sap2010:WorkflowViewState.IdRef="LogMessage_3" Level="Info" Message="[&quot;Main started&quot;]" />
+          <ui:LogMessage DisplayName="Log Message - Main" sap:VirtualizedContainerService.HintSize="334,91" sap2010:WorkflowViewState.IdRef="LogMessage_3" Level="Info" Message="[&quot;Main started..&quot;]" />

```

Comment:
Addnotation on main sequence (brief summary of the project)
Please update project description
Not necessary to rename all invoked wfs. Rename them if the appear 2 or more times
You can group wfs and invoke them in an another wf (that will be invoked in Main, in your case). It's easier for unit testing and will be not necessary to creat a test case for each wf if you're grouping wfs
Hardcoded delays in SearchSquadPlayer, Click add player (+). Usually delays are included in Config file (next projects ;) )
Worng argument type in SelectLeagues, out_leagueChoice. You're using this arg as "IN"
Spaces in selectors (e.g.aaname 'Leagues '). There are many. This is kinda unstable because may change in the future (may be same name but not using spaces). Please try to replace. This may be a differece between test and prod environments and a common change.
SquadBuilder, Find squad positions selector title is 'FIFA 22 Squad Builder | FUTBIN'
E.g. SearchSquadPlayer, Click player using index to identify selector. An observation, but when is possiblie try not to use index in selectors
It's ok that you verified that site is already open. If site already open, close it because you close application at the end of the process
Folder "Futbin" should be in an another folder named "ApplicationLayer"
Folder "Test Cases" should not contain space in the name (when downloading a package from orch, special characters are replaced in unicode)
In logs at the start of each wf, try to put the message as "Start WfName"
SendEmail wf argument in_squadUrl should be changed with in_bodyEmail, for example, but try to compose a message. Try no to set email subject as "Business Rule Exception". Instead mail subject could be "Unknown error".